deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -560,12 +586,47 @@ private void addDescription(ContentObject obj) throws IOException {
 	}
 	
 	private void addPremisEvents(ContentObject obj) {
-		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".xml");
+		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".ttl");
 		if (!premisFile.exists()) {
 			return;
 		}
-		FilePremisLogger premisLogger = new FilePremisLogger(obj.getPid(), premisFile, repository);
+		PremisLogger premisLogger = getPremisLogger(obj.getPid());
 		List<PremisEventObject> events = premisLogger.getEvents();
+		// add premis events to fedora
 		obj.addPremisEvents(events);
 	}
+	
+	private void addIngestionEventForContainer(ContentContainerObject obj, Model model) throws IOException {
+		PremisLogger premisLogger = getPremisLogger(obj.getPid());
+		PremisEventBuilder builder = premisLogger.buildEvent(Premis.Ingestion);
+		builder.addEventDetail("ingested as PID: {0}", obj.getPid().toString());
```

Comment:
The container should only ever get one event with this message, the first time it is ingested.  I believe at the moment the destination container would get a new one of these everytime an ingest happens in it.